src/index.js

```diff
@@ -178,14 +210,59 @@ const moduleToApp = (ddoc, module) =>
     digest: ddoc._attachments[module].digest,
   });
 
+const deployDdoc = stagedDdoc => {
+  info(`Deploy staged ddoc ${stagedDdoc._id} to ${DDOC}`);
+
+  const deletedStagedDdoc = {
+    _id: stagedDdoc._id,
+    _rev: stagedDdoc._rev,
+    _deleted: true
+  };
+
+  info('Getting currently live DDOC');
+  return db.get(DDOC)
+    .catch(err => {
+      if (err.status !== 404) {
+        throw err;
+      }
+
+      info('No existing live DDOC');
```

Comment:
Might be helpful to make this a `warn`, although I see that's it's not necessarily a negative.